package.json

```diff
@@ -10,7 +10,7 @@
     "build": "astro check && astro build",
     "preview": "astro preview",
     "astro": "astro",
-    "build:contribute": "tsx ./src/scripts/build-contribute.ts",
+    "build:contribute": "tsx ./src/scripts/builders/contribute.ts",
```

Comment:
haha looks like this naming pattern won out in the end by circumstance. I was always thinking I would try to find time to return to reconsider the script names and organization but now that there is uniformity might as well just let sleeping dogs lie